### PR TITLE
Add re-configuration item to troubleshooting guide.

### PR DESCRIPTION
**Pull request by duftler**
@tomas please review.

### CODE REVIEW

File: documentation/troubleshooting.md [hunk 1]
```diff
@@ -28,3 +28,21 @@ Thrift should now be enabled. Execute `curl localhost:9160` and verify that you
...
+There are various ways you can modify your configuration:
+* Re-running `InstallSpinnaker.sh`
+* Editing `/etc/default/spinnaker`
+* Editing one of the `.yml` files (e.g. `/opt/spinnaker/config/spinnaker-local.yml`, `/opt/spinnaker/config/clouddriver.yml`, `/opt/rosco/config/rosco.yml`)
+* Modifying environment variables
+* Modifying `~/.aws/credentials`
+
+If you've modified your configuration via any of those methods, the simplest way to have Spinnaker synchronize your configuration is to run these two commands:
+
+\# Restart all Spinnaker subsystems
```
Comment (tomaslin) [inline documentation/troubleshooting.md]: should probably document how to restart just one service

File: documentation/troubleshooting.md [hunk 2]
```diff
@@ -28,3 +28,21 @@ Thrift should now be enabled. Execute `curl localhost:9160` and verify that you
 The last step is to restart the two Spinnaker services that require Cassandra to be available on startup: `sudo service front50 restart` and `sudo service echo restart`.
 
 We will be making front50 and [echo](https://github.com/spinnaker/echo) more tolerant of an unavailable or misconfigured Cassandra cluster on startup shortly.
+
+## I changed my configuration. How do I get Spinnaker to pick up the modified configuration?
+There are various ways you can modify your configuration:
```
Comment (tomaslin) [inline documentation/troubleshooting.md]: would be good to make it explicit that this is for the aws or gce on instance installation

File: documentation/troubleshooting.md (hunk 1 above)
Comment (tomaslin) [inline documentation/troubleshooting.md]: link to clouddriver's refresh config endpoint

File: documentation/troubleshooting.md (hunk 2 above)
Comment (duftler) [inline documentation/troubleshooting.md]: Do you mean as opposed to running locally for development?

Comment (tomaslin) [inline documentation/troubleshooting.md]: yes.

**Pull request merged**